BaseTools/Source/C/GenVtf/GenVtf.c

```diff
@@ -129,9 +129,9 @@ Routine Description:
   } else {
     Length = strlen(Str);
     if (Length < 4) {
-      strncpy (TemStr + 4 - Length, Str, Length);
+      memcpy (TemStr + 4 - Length, Str, Length);

```

Comment:
The issue here was that the copy should be limited by the target not by the source, I'm not sure if for the current solution there is no possibility of overriding '\0' (end of string)